rocAL/source/readers/image/file_source_reader.cpp

```diff
@@ -166,7 +167,7 @@ Reader::Status FileSourceReader::subfolder_reading() {
                 std::string file_extension = subfolder_path.substr(file_extension_idx + 1);
                 std::transform(file_extension.begin(), file_extension.end(), file_extension.begin(),
                                [](unsigned char c) { return std::tolower(c); });
-                if ((file_extension != "jpg") && (file_extension != "jpeg") && (file_extension != "png") && (file_extension != "ppm") && (file_extension != "bmp") && (file_extension != "pgm") && (file_extension != "tif") && (file_extension != "tiff") && (file_extension != "webp"))
+                if ((file_extension != "jpg") && (file_extension != "jpeg") && (file_extension != "png") && (file_extension != "ppm") && (file_extension != "bmp") && (file_extension != "pgm") && (file_extension != "tif") && (file_extension != "tiff") && (file_extension != "webp") && (file_extension != "wav"))
```

Comment:
Why are we adding "wav" file to an image reader file?

Reply:
Please make sure the header paths are changed wherever this file is used to reflect this change